File: doc/api/ruby/example_about.rb
#!/usr/bin/env ruby
# vim: set sw=4 sts=4 et tw=80 :

=begin description
Simple example showing how to use Paludis version constants.
=end

require 'Paludis'

print "Built using Paludis ", Paludis::VersionMajor, ".", Paludis::VersionMinor,
    ".", Paludis::VersionMicro, Paludis::VersionSuffix

if not Paludis::GitHead.empty?
    print ' ', Paludis::GitHead
end

print "\n"

File: doc/api/python/example_about.py
#!/usr/bin/env python
# vim: set fileencoding=utf-8 sw=4 sts=4 et :

"""A simple example showing how to use Paludis version constants"""

import paludis

print("Built using Paludis " + str(paludis.VERSION_MAJOR) + "." + str(paludis.VERSION_MINOR) +
        "." + str(paludis.VERSION_MICRO) + paludis.VERSION_SUFFIX),

if paludis.GIT_HEAD:
    print 'git-' + paludis.GIT_HEAD
